scripts/image-builder/configure.sh

```diff
@@ -28,6 +28,15 @@ if grep -qE "Red Hat Enterprise Linux.*Beta" /etc/redhat-release; then
    sudo sed -i "s,dist/rhel9/${VID},beta/rhel9/9,g" /usr/share/osbuild-composer/repositories/rhel-"${VID}".json
 fi
 
+# Following edits composer's configuration to enable RT repository.
+# Duplicate first repository (baseos), change its name, and replace 'baseos' with 'rt'.
+# kernel-rt is only available for x86_64.
```

Comment:
What's happening on arm64 platform for RT kernel?

Reply:
Nothing, there's no rt kernel for arm

Reply:
If you're asking what the status of the development to bring RT to arm, then I don't know

Reply:
OK, I see. Let's resolve this comment.